include/glow/LLVMIRCodeGen/LLVMCompiledFunction.h

```diff
@@ -20,8 +20,25 @@
 
 #include "glow/Backend/BackendUtils.h"
 #include "glow/Backend/CompiledFunction.h"
+#include "glow/ExecutionContext/ExecutionContext.h"
 
 namespace glow {
+
+/// CPUDeviceBindings inherits from DeviceBindings, it contains per run
```

Comment:
Seeing as how this is used in llvmCompiledFunction I'd just name it llvmDeviceBindings.

Reply:
Done. I also added CPUDeviceBindings just to keep clear naming and structure when used by CPUDeviceManager.